### PR TITLE
WX-1307 Azure E2E test

### PR DESCRIPTION
**Pull request by JVThomas**
This draft PR is one half (WIP) of migrating the E2E workflow test over to the dsp-reusable-workflows repo. There is a sibling branch on that repo as well that also needs to be developed along side this (same branch name [`WX-1307-port`], but no PR made just yet.)

The PR here simply reduces the workflow so that the run-script job simply generates a token and passes all required inputs to the test script workflow housed in the other repo.

If this migration moves forward, the sibling branch must either be merged first or merged at the same time as this one to avoid having to worry about branch referencing in the workflow.

If migrating is optional, then you can choose to move forward or drop it if other priorities pop up (since the vanilla `WX-1307` branch can be used to cover the e2e testing from the Cromwell repo).

### CODE REVIEW

File: .github/workflows/run_workflow.yml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  schedule:
+    - cron: '0 16 * * *' # UTC 4pm, EST 11am, EDT 12pm
+  workflow_dispatch:
+    inputs:
+      target-branch:
+        description: 'Branch name of dsp-reusable-workflows repo to run tests on'
+        required: true
+        default: 'main'
+        type: string
+      # Replace user data with seeded data provided by devOps (once available)
```
Comment (salonishah11) [inline .github/workflows/run_workflow.yml:12]: small nit: Can this comment be removed now?

File: .gitignore [hunk 2]
```diff
@@ -55,3 +58,6 @@ tesk_application.conf
 **/venv/
 exome_germline_single_sample_v1.3/
 **/*.pyc
+
+# GHA credentials
+gha-creds-*.json
```
Comment (salonishah11) [inline .gitignore:63]: Is this and `bloop` being used anywhere? 🤔

**Review by salonishah11 — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (jgainerdewar) [reply .gitignore:63]: I think some folks on the team use bloop for Scala builds. I assume gha-creds is for being able to test things locally?

Comment (jgainerdewar) [reply .gitignore:63]: Not used in this PR directly, I assume Justin added these during his development work.

**Review by jgainerdewar — APPROVED** [review]

**Comment by jgainerdewar** [comment]
GCP Batch backend failure doesn't block merging.

**Pull request merged**